pages/concludingStudents.vue

```diff
@@ -1,28 +1,101 @@
 <template>
   <div class="container">
+    <div class="columns is-centered">
+      <div class="column is-5 is-offset-2">
+        <h1 class="title">
+          <strong>Alunos Concluintes</strong>
+        </h1>
+      </div>
+    </div>
+    <div class="columns">
+      <div class="column is-3 is-offset-8 to-left">
+        <b-button class="is-primary" @click="getAcademicHighlights">
+          Gerar lista de destaques acadêmicos
+        </b-button>
+      </div>
+    </div>
     <search-input
       :key="courseTag"
       :default-course="courseTag"
-      :title="'Alunos Concluintes'"
       :is-concluding="1"
     ></search-input>
+    <b-modal :active.sync="activeModal">
+      <div class="card">
+        <header class="card-header">
+          <b-icon pack="fas" icon="star" size="is-medium"></b-icon>
+          <p class="card-header-title">Alunos destaque acadêmico:</p>
+        </header>
+        <div class="card-content">
+          <div class="content">
+            <table>
+              <thead>
+                <th scope="col">Nome</th>
+                <th scope="col">Período</th>
+              </thead>
+              <tbody>
+                <tr v-for="(student, id) in highlighted" :key="id">
+                  <td>{{ student.name }}</td>
+                  <td>{{ student.period || '-' }}</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+    </b-modal>
   </div>
 </template>
 
 <script>
 import { mapState } from 'vuex'
 import SearchInput from '~/components/searchInput'
+import { errorsHandler } from '../components/mixins/errors'
 
 export default {
   name: 'Concluding',
   middleware: 'course',
   components: {
     SearchInput
   },
+  mixins: [errorsHandler],
+  data: () => ({
+    activeModal: false,
+    highlighted: []
+  }),
```

Comment:
não seria melhor utilizar `data(){
return{
activateModal:false,
highlighted:[]
} 
}` ao inves de uma arrow function ?

Reply:
Why not 🤷

Reply:
pronto, mudado